### PR TITLE
[CLEANUP] Simplify RenderingDispatcher::render

### PR DESCRIPTION
**Pull request by oliverklee**

### CODE REVIEW

File: Classes/ViewHelpers/Widget/LinkViewHelper.php [hunk 1]
```diff
@@ -67,13 +67,10 @@ public function initializeArguments()
      */
     public function render($pluginName, $extensionName, $action = null, $arguments = array(), $section = '', $format = '', $ajax = true)
     {
-        if ($ajax === true) {
-            $uri = $this->getAjaxUri();
-        } else {
-            $uri = $this->getWidgetUri();
-        }
+        $uri = $ajax ? $this->getAjaxUri() : $this->getWidgetUri();
```
Comment (helhum) [inline Classes/ViewHelpers/Widget/LinkViewHelper.php]: This change makes the code to behave differently. besides that, tenary operators do not improve the code by default.

Comment (oliverklee) [inline Classes/ViewHelpers/Widget/LinkViewHelper.php]: In which way is the behavior different?

Comment (helhum) [inline Classes/ViewHelpers/Widget/LinkViewHelper.php]: When not taking into consideration that Fluid correctly casts this variable to a boolean, then explicitly asking for a boolean true is different than assuming true for all other possible values PHP casts to be true ('1', 1, array('asd'), new \stdClass, …)

Comment (oliverklee) [inline Classes/ViewHelpers/Widget/LinkViewHelper.php]: Ah, okay. Thanks for the clarification.

So Fluid does not ensure the corresponding type?

Comment (helhum) [inline Classes/ViewHelpers/Widget/LinkViewHelper.php]: > So Fluid does not ensure the corresponding type?

Yes, Fluid does so. Nevertheless that is not clear when reading these specific lines of code.

**Pull request closed without merging**